anxcloud/testutils/environment/naming.go

```diff
@@ -0,0 +1,178 @@
+package environment
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+var (
+	adjectives = [...]string{
+		"eager",
+		"easy",
+		"elated",
+		"elegant",
+		"embarrassed",
+		"enchanting",
+		"encouraging",
+		"energetic",
+		"enthusiastic",
+		"envious",
+		"evil",
+		"excited",
+		"expensive",
+		"exuberant",
+		"fair",
+		"faithful",
+		"famous",
+		"fancy",
+		"fantastic",
+		"fierce",
+		"filthy",
+		"fine",
+		"foolish",
+		"fragile",
+		"frail",
+		"frantic",
+		"friendly",
+		"frightened",
+		"funny",
+		"gentle",
+		"gifted",
+		"glamorous",
+		"gleaming",
+		"glorious",
+		"good",
+		"gorgeous",
+		"graceful",
+		"grieving",
+		"grotesque",
+		"grumpy",
+		"handsome",
+		"happy",
+		"healthy",
+		"helpful",
+		"helpless",
+		"hilarious",
+		"homeless",
+		"homely",
+		"horrible",
+		"hungry",
+		"hurt",
+		"ill",
+		"important",
+		"impossible",
+		"inexpensive",
+		"innocent",
+		"inquisitive",
+		"itchy",
+		"jealous",
+		"jittery",
+		"jolly",
+		"joyous",
+		"kind",
+	}
+	nouns = [...]string{
+		"sort",
+		"boot",
+		"cap",
+		"square",
+		"bucket",
+		"hat",
+		"bed",
+		"lip",
+		"hand",
+		"toys",
+		"pollution",
+		"summer",
+		"visitor",
+		"observation",
+		"question",
+		"route",
+		"truck",
+		"cushion",
+		"plant",
+		"dolls",
+		"partner",
+		"bag",
+		"use",
+		"army",
+		"plate",
+		"cable",
+		"connection",
+		"screw",
+		"fairies",
+		"notebook",
+		"insurance",
+		"pen",
+		"elbow",
+		"holiday",
+		"rock",
+		"horn",
+		"building",
+		"kitty",
+		"jeans",
+		"sisters",
+		"bird",
+		"berry",
+		"fang",
+		"metal",
+		"aunt",
+		"treatment",
+		"bait",
+		"chess",
+		"wave",
+		"coat",
+		"brother",
+		"color",
+		"tent",
+		"mine",
+		"wealth",
+		"marble",
+		"secretary",
+		"zinc",
+		"grip",
+		"control",
+		"respect",
+		"clover",
+		"home",
+		"seat",
+		"donkey",
+		"measure",
+		"cellar",
+		"governor",
+		"drop",
+		"dock",
+		"week",
+		"verse",
+		"robin",
+		"lake",
+		"station",
+		"spiders",
+		"peace",
+		"railway",
+		"geese",
+		"playground",
+		"horses",
+		"blow",
+		"care",
+		"jail",
+		"pot",
+		"noise",
+		"children",
+		"shop",
+		"salt",
+		"reaction",
+		"crate",
+		"spark",
+		"branch",
+		"shame",
+		"need",
+		"cheese",
+		"laborer",
+		"love",
+	}
+)
+
+func generateRandomName() string {
```

Comment:
I'm ok with this code, but would've expected a library like [github.com/goombaio/namegenerator](https://pkg.go.dev/github.com/goombaio/namegenerator) be used instead - is there a reason why you don't use a library?

Reply:
not really. i don't really have an opinion about this one. I just thought since it's just a function i could simply write it by myself and don't have to add a dependency which is just used for a test. for the binary in the end it doesn't matter whether its a lib or self-written. I also liked the idea of being in control over the adjective and nouns to make sure it doesn't contain words that could trigger anyone. however looking at the library you brought, these words seem to be well chosen. So I will just use it